### ISSUE TITLE
setuptool deprecation triggering pre-commit error (already fixed on main)

### ISSUE THREAD

**Issue by RonnyPfannschmidt** [AleksaC/hadolint-py#4]
```
Failed to build hadolint-py
stderr:
      error: subprocess-exited-with-error
      
      × python setup.py bdist_wheel did not run successfully.
      │ exit code: 1
      ╰─> [19 lines of output]
          /home/ronny/.cache/pre-commit/repoyxbcuv9c/py_env-python3.11/lib/python3.11/site-packages/setuptools/config/setupcfg.py:293: _DeprecatedConfig: Deprecated config in `setup.cfg`
          !!
          
                  ********************************************************************************
                  The license_file parameter is deprecated, use license_files instead.
          
                  By 2023-Oct-30, you need to update your project and remove deprecated calls
                  or your builds will no longer be supported.
          
                  See https://setuptools.pypa.io/en/latest/userguide/declarative_config.html for details.
                  ********************************************************************************
          
          !!
            parsed = self.parsers.get(option_name, lambda x: x)(value)
```

i believe the cause is the lack of a new tag with ed40aa186f6d46a5c74b59c9917bfc56f67edcbd

**Comment by RonnyPfannschmidt** [comment]
a workaround is to use a old python for the hook

**Comment by AleksaC** [comment]
Alternatively you could have used the `ed40aa1` sha directly as a rev. There's no need for that anymore as I've released the fix with [`v2.12.0.3` tag](https://github.com/AleksaC/hadolint-py/releases/tag/v2.12.0.3).
